pkg/controllers/dynakube/extension/consts/consts.go

```diff
@@ -0,0 +1,19 @@
+package consts
```

Comment:
Just want to point out that in other packages inside `pkg/controllers/dynakube` we don't create a subpackage for the consts (e.g. `pkg/controllers/dynakube/proxy/consts.go`).
We should keep the same package structure in the directory IMO

So either rolling back this change, or updating the existing consts.go files to its own subdir.

Reply:
`consts` sub-directory / package is needed because otherwise can't import them from `extension/eec` , `extension/otel` packages because of circular dependency
```
extension <- extension/eec <- extension
```

Reply:
billions of messages to be solved by newer EEC image or a configuration file